.gitignore

```diff
@@ -1,20 +1,27 @@
-/CMake*
-!/CMakeLists.txt
-/*.cmake
-/Makefile
-/gmock/
-/*.cpp
-/include/
-/ts/ts.qrc.depends
-/chewing-editor.desktop
-/chewing-editor
-/chewing-editor.1
-/chewing-editor.1.gz
-/chewing.json
-/*.a
-/run-test*
-/ts/*.qm
-/ui_*.h
-/Testing/
-/coverage*
-/chewing-editor.dSYM/
+CMake*
+!CMakeLists.txt
+*.cmake
+Makefile
+
+gmock/
+
+*moc*
+qrc*
+ts.qrc.depends
+
+chewing-editor
+chewing-editor.exe
+chewing-editor.desktop
+chewing-editor.1*
+
+config.h
+ui_*.h
+*.json
+*.a
+run-test*
+
+*.qm
+
+coverage*
+
+chewing-editor.dSYM/
```

Comment:
@david50407 

I changed this one, removed the root path `/`, can you confirm this will work?
If not, I'll revert.